mmdet/models/roi_heads/mask_heads/fcn_mask_head.py

```diff
@@ -288,7 +288,7 @@ class label c.
             # GPU benefits from parallelism for larger chunks,
             # but may have memory issue
             num_chunks = int(
-                np.ceil(N * img_h * img_w * BYTES_PER_FLOAT / GPU_MEM_LIMIT))
+                np.ceil(N * int(img_h) * int(img_w) * BYTES_PER_FLOAT / GPU_MEM_LIMIT))
```

Comment:
We may refer to this PR in the comment and explain why `int` is necessary here, thus the readers of the code could easily understand the motivation.